tests/rkt_exit_test.go

```diff
@@ -71,13 +71,16 @@ func TestExitCodeWithSeveralApps(t *testing.T) {
 	}
 
 	t.Logf("Waiting pod termination\n")
-	waitOrFail(t, child, true)
+	// Currently we have systemd v222 in the coreos flavor which doesn't
+	// include the exit status propagation code.
+	// TODO(iaguis): we should expect 5 as the exit status when we update to v229
+	waitOrFail(t, child, 0)
```

Comment:
Would this break tests with `tests/aws.sh` on Fedora-Rawhide with the host flavor (it should have the last systemd)?